From 4a315d773daa508d46e0246d78642fce6d71287e Mon Sep 17 00:00:00 2001
From: Haikel Guemar <hguemar@fedoraproject.org>
Date: Tue, 14 Oct 2014 15:30:36 +0200
Subject: [PATCH 2/2] Remove runtime dep on pbr

---
 manila/version.py | 22 ++++++++++++++++++++--
 1 file changed, 20 insertions(+), 2 deletions(-)

diff --git a/manila/version.py b/manila/version.py
index 75aecef..9e912a8 100644
--- a/manila/version.py
+++ b/manila/version.py
@@ -14,12 +14,30 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-from pbr import version as pbr_version
 
 MANILA_VENDOR = "OpenStack Foundation"
 MANILA_PRODUCT = "OpenStack Manila"
 MANILA_PACKAGE = None  # OS distro package version suffix
 
+class VersionInfo(object):
+    release = "REDHATMANILARELEASE"
+    version = "REDHATMANILAVERSION"
+
+    def version_string(self):
+        return self.version
+
+    def cached_version_string(self):
+        return self.version
+
+    def release_string(self):
+        return self.release
+
+    def canonical_version_string(self):
+        return self.version
+
+    def version_string_with_vcs(self):
+        return self.release
+
 loaded = False
-version_info = pbr_version.VersionInfo('manila')
+version_info = VersionInfo()
 version_string = version_info.version_string
-- 
2.1.0

